libs/blocks/icon-block/icon-block.js

```diff
@@ -39,6 +39,13 @@ const iconBlocks = {
   },
 };
 
+//checking if block is part of xx-up and is inline variant
+function upAndInline(el) {
+  const isInlineAndSectionMetadata = el.classList.contains('inline') && el.parentElement?.querySelector('.section-metadata')
+  if(!isInlineAndSectionMetadata) return false;
```

Comment:
nit: our lint standard requires a space between if and (
```suggestion
  if (!isInlineAndSectionMetadata) return false;
```